make/prepare

```diff
@@ -23,9 +33,6 @@ mkdir -p $input_dir/common
 cp $ssl_cert_path $input_dir/nginx/server.crt
 cp $ssl_cert_key_path $input_dir/nginx/server.key
 
-# Copy secretkey to input dir
-cp -r $secretkey_path $input_dir/keys
-
 # Copy ca bundle to input dir
 if [ -f $registry_custom_ca_bundle ]
```

Comment:
please pass the image tag($1) into prepare rather than hard-coding it to `goharbor/prepare:dev`